src/sortedSteps.ts

```diff
@@ -63,6 +64,14 @@ export async function sortedSteps(
         const dependency = await getAndCacheDependency(
           potentialEffectDependency,
         );
+
+        // Accept all conditions/steps regardless
+        if (acceptAllConditions) {
```

Comment:
can we assert in the unit test that the `.accept` function of a conditional is not called at al with `acceptAllConditions = true`? We want to make sure that we don't waste cycles calculating a result that is never used, some of the acceptors can do quite some heavy lifting.